lib/beaker/cli.rb

```diff
@@ -103,8 +116,6 @@ def execute!
           exit(1)
         end
 
-        provision
```

Comment:
@ki I took the `provision` call out of execute, had it return self, and then just added it to the invocation in `bin/beaker`. I think this was a good thing, as it separates `provisioning` and `execution` more cleanly. With this method, however, we can't call things like `configure` and `validate` and thus shouldn't be able to accept those flags.

Alternatively, we could modify provision to change some of the `SUBCOMMAND_OPTIONS` and set it options there to not `validate` or `configure`. If done that way, we can keep `provision` in the `execute!` method.

@ferglor 's destroy might have an opinion, as it seems very intertwined what provision does...